templates/Includes/Carousel.ss

```diff
@@ -72,18 +72,16 @@
                             <% end_if %>
 
                             <% if $PrimaryCallToActionLabel || $SecondaryCallToActionLabel %>
-                                <p>
-                                    <% if $PrimaryCallToActionLabel %>
-                                        <a class="btn btn-primary btn-lg" href="{$PrimaryCallToAction.Link}">
-                                            $PrimaryCallToActionLabel
-                                        </a>
-                                    <% end_if %>
-                                    <% if $SecondaryCallToActionLabel %>
-                                        <a class="btn btn-lg btn-link" href="{$SecondaryCallToAction.Link}">
-                                            $SecondaryCallToActionLabel
-                                        </a>
-                                    <% end_if %>
-                                </p>
+                                <% if $PrimaryCallToActionLabel %>
+                                    <a class="btn btn-primary btn-lg" href="{$PrimaryCallToAction.Link}">
+                                        $PrimaryCallToActionLabel
+                                    </a>
+                                <% end_if %>
+                                <% if $SecondaryCallToActionLabel %>
+                                <a class="btn btn-lg btn-outline-light btn-secondary-action" href="{$SecondaryCallToAction.Link}">
```

Comment:
Perhaps `carousel-secondary-action` would make a little more sense here? Unlike the other `btn` modifiers, `btn-secondary-action` does not providing global styling and is very specific to the carousel.